octodns_dnsmadeeasy/__init__.py

```diff
@@ -236,7 +243,15 @@ def _data_for_CAA(self, _type, records):
         return {'ttl': records[0]['ttl'], 'type': _type, 'values': values}
 
     def _data_for_TXT(self, _type, records):
-        values = [value['value'].replace(';', '\\;') for value in records]
+        # Long TXT records in DNS Mady Easy have their value split into 255 character chunks, delimited by "".
+        values = [
+            re.sub(
+                self.TXT_RECORD_VALUE_DELIMITER_PATTERN,
+                '',
+                value['value'].replace(';', '\\;'),
+            )
```

Comment:
I believe that's passing the compiled regex into the sub each time, and I presume creating a new one. based on the pattern. The `sub` method is available on the compiled regex. 

```suggestion
            self.TXT_RECORD_VALUE_DELIMITER_PATTERN.sub(
                '',
                value['value'].replace(';', '\\;'),
            )
```

Reply:
Sorry, I forgot to update it to use the compiled regex.